src/modules/user/kyc/addresses/addresses.test.ts

```diff
@@ -0,0 +1,99 @@
+import MockAdapter from 'axios-mock-adapter';
```

Comment:
Please, move this file to sagas folder and rename it to <saga name>.test.ts

Reply:
good suggestion, will be changed